www/content/tutorial.md

```diff
@@ -1037,6 +1037,14 @@ The `*` is a _wildcard type_; a type that's compatible with any other type. `Lis
 
 The wildcard type also comes up with empty lists. Suppose we have one function that takes a `List Str` and another function that takes a `List Bool`. We might reasonably expect to be able to pass an empty list (that is, `[]`) to either of these functions, and we can! This is because a `[]` value has the type `List *`. It is a "list with a wildcard type parameter", or a "list whose element type could be anything."
 
+Let's test your understanding, for a function with the signature `List * -> List *`, what kind of arguments would it accept and what would it return? Take a moment to think.
+
+- The function could only return `[]`. Remember, the wildcard type means **any** type, so our function `List * -> List *` **must** return a list that can contain **any** type. The only specific list that would fit the description is the empty list. 
+- Note the function would not only accept `[]` as argument, but any list!
+Function arguments are allowed to be broad/general but the values you return must be specific.
```

Comment:
@rtfeldman I feel like this sentence is important but I'm not sure this is the best way to put it.